### PR TITLE
AOM-136: fix deployment issues

### PR DESCRIPTION
**Pull request by patlub**
## JIRA TICKET NAME:

[AOM-36: fIx deployment issues](https://issues.openmrs.org/browse/AOM-136)

### SUMMARY:

The application does not deploy to Bintray successfully. This task fixes those issues

### CODE REVIEW

File: bintray.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
     "package": {
-        "name": "OpenMRS-Add-On-Manager",
-        "repo": "Apps",
+        "name": "Add-On-Manager",
```
Comment (dkayiwa) [inline bintray.json]: Do you have any reason for using hyphens in the name?

File: bintray.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
     "package": {
-        "name": "OpenMRS-Add-On-Manager",
```
Comment (dkayiwa) [inline bintray.json:3]: The package should be openmrs-owa-addonmanager

**Comment by dkayiwa** [comment]
There are merge conflicts

File: bintray.json (hunk 2 above)
Comment (patlub) [reply bintray.json:3]: Fixed

**Review by aenkya — APPROVED** [review]
Looks good to me

**Pull request merged**